### PR TITLE
Add CLI methods create, add_public_key

### PR DESCRIPTION
**Pull request by zorkian**
This allows us to both create users and add public keys for users from
the command line.

Also fix a bug where deletes were logged on the wrong account.

### CODE REVIEW

File: bin/grouper-ctl [hunk 1]
```diff
@@ -194,6 +228,15 @@ def main():
     user_parser.set_defaults(func=user_command)
     user_subparser = user_parser.add_subparsers(dest="subcommand")
 
+    user_create_parser = user_subparser.add_parser(
```
Comment (tmildorf) [inline bin/grouper-ctl]: I'm having trouble getting this help info for sub commands to display. I've tried grouper-ctl --help, grouper-ctl help, grouper-ctl --help sync_db, grouper-ctl help sync_db, grouper-ctl sync_db --help; am I missing the right permutation?

**Comment by tmildorf** [comment]
$ bin/grouper-ctl -v -c config/dev.yaml user add_public_key alex@example.com AAAA**SNIP**==
No such user alex@example.com

File: bin/grouper-ctl (hunk 1 above)
Comment (zorkian) [inline bin/grouper-ctl]: Just type whatever and append `-h`:

```
bin/grouper-ctl user -h
bin/grouper-ctl user set_metadata -h
```

etc.

**Comment by zorkian** [comment]
Did you create the user? If the user doesn't exist, then you can't set a key on them...

**Comment by tmildorf** [comment]
I succeeded by using two separate commands.

**Comment by zorkian** [comment]
Yeah; `create` is the only way to create a user. I wanted it to be an explicit command, rather than having someone typo a username when trying to add a public key.

**Comment by tmildorf** [comment]
My impression from skimming the code was that I should be able to figure out how to specify both `create` and `add_public_key` subcommands but that may not be supported.  It's a minor point.  I approve.

**Comment by tmildorf** [comment]
Making the help a little easier to get through can be addressed separately.  The desired functionality is the key here.

**Pull request merged**